src/input/RotaryInputAdapter.h

```diff
@@ -0,0 +1,72 @@
+#pragma once
+
+#include "InputInterface.h"
+#include <SimpleRotary.h>
+
+class RotaryInputAdapter : public InputInterface {
```

Comment:
Maybe better to have the same name as used component? Like `SimpleRotaryAdapter`.